pkgs/development/ocaml-modules/eio/default.nix

```diff
@@ -18,14 +18,14 @@
 
 buildDunePackage rec {
   pname = "eio";
-  version = "0.12";
+  version = "0.13";
 
   minimalOCamlVersion = "5.0";
```

Comment:
This is no longer true. And that breaks `wayland-proxy-virtwl`: you may consider keeping version 0.12 around for compatibility with OCaml 5.0.

Reply:
ouch

Reply:
Can we try this??